admin_guide/manage_users.adoc

```diff
@@ -14,19 +14,68 @@ toc::[]
 [[managing-users-overview]]
 == Overview
 
+A user is an entity that interacts with the {product-title} API. These can be a developer for developing applications or a administrator for managing the cluster. Users can be assigned to groups, which set the permissions applied to all the group's members. For example, you can give API access to a group, which give all members of the group API access.
+
 This topic describes the management of
-xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user] accounts,
-including how new user accounts are created in {product-title} and how they can
-be deleted.
+xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user]
+accounts, including how new user accounts are created in {product-title} and how
+they can be deleted.
 
 [[managing-users-adding-a-user]]
-== Adding a User
+== Creating and Adding a User
+
+The
+xref:../install_config/configuring_authentication.adoc#identity-providers[identity
+provider] configured on the master, dictates how to create a user. For example,
+if the identity provider is set to `AllowAll`, then the user simply uses `oc
+login` and enters their user name and password at the prompts.
+
+The following process is to create a new user as a system administrator, then
+add a role to the user:
+
+. Create the user account:
++
+----
+# touch /path/to/identityproviers/file /
+  <identity_provider> -b /path/to/identityproviers/file /
+  <username> <password>
+----
```

Comment:
This does not make sense as not every authentication provide has a command that follows this syntax!

Reply:
This also does not apply to the AllowAll provider. 

~~~
oc login -u USER
~~~

Is a better example for this.